tests/uefistored/test_auth_var.py

```diff
@@ -83,14 +81,16 @@ def test_db_append(linux_uefi_vm):
 
     assert vm.ssh_with_result(["which", "efi-updatevar"]).returncode == 0, "This test requires efi-updatevar"
 
+    old = vm.get_efi_var(db.name, db.guid)
+
+    assert old != b"", "db failed to install"
```

Comment:
I think this assert is always true due to the change you made to the get_efi_var function.

Reply:
If `/sys/firmware/efi/efivars/db-GUID` is empty, this will fire (common.py:936). This happened to me once, but unlikely to happen again. I think had accidentally used the wrong GUID with the old API, if I'm remember correctly. I think it can safely be removed.